mainwindow.py

```diff
@@ -779,7 +779,7 @@ def toggle_bom_pos(self, *_):
             fp = board.FindFootprintByReference(ref)
             bom = toggle_exclude_from_bom(fp)
             pos = toggle_exclude_from_pos(fp)
-            self.store.set_bom(ref, bom)
+            self.store.set_bom(ref, int(bom))
```

Comment:
what was up with the necessity to cast here but not previously? (No mention in the commit message of the reason why)

Reply:
I don't remember but I guess that sqlite doesn't like to get a "1" for a boolean value. Previously the casting was done in th e set_bom function, in other places it was done like this. So I decided to make the cast always in the same place.